model/browser/events.yaml

```diff
@@ -0,0 +1,82 @@
+groups:
+  - id: event.browser.user_action
+    stability: development
+    type: event
+    name: browser.user_action
```

Comment:
Do we think there's an opportunity to use another prefix/namespace other than `browser`? It seems like this is something that mobile (ios/android) could certainly also benefit from as well. If we can't make it broader to support multiple clients, then I fear we will encounter duplication later. I also undertand if the answer is "no, it doesn't make sense because browser already has all this stuff (eg. element xpath below, etc)"...but I think we should at least consider it.

Possible alternative namespaces: `client.user_action`, `app.user_action`, `

Reply:
Maybe, but that is a larger discussion for the Client SIG. We can always deprecate this event in favor of any stable standard. But generally it would be "nice" to have to define a commonly used event once -- but there are some very browser centric definitions here at the moment.

Reply:
enduser.user_action

Reply:
@Karlie-777 @breedx-splk it looks like there are still several browser centric definitions in this PR.

I think it will need a good amount of help from the Android folks to generalize this.

Otherwise I'd recommend landing it as a browser event and consider generalizing it as a follow-up.

As @MSNev says

> We can always deprecate this event in favor of any stable standard.

Reply:
> Otherwise I'd recommend landing it as a browser event and consider generalizing it as a follow-up.

I'm into this. And because I opened this thread I'm happy to resolve it...I don't think it should block.